### ISSUE TITLE
Feedback for: Introduction to Push Notifications [en]

### ISSUE THREAD

**Issue by nasearle** [google/WebFundamentals#4262]
https://github.com/google/WebFundamentals/blob/master/src/content/en/ilt/pwa/introduction-to-push-notifications.md

Is there a way to force these code blocks to wrap instead of using side-scrollers?

![screen shot 2017-02-24 at 1 24 58 pm](https://cloud.githubusercontent.com/assets/14795039/23321552/df868588-fa94-11e6-866b-c0d1a594f681.png)

**Comment by petele** [comment]
Yes. The typical syntax is to break lines with a `\`. So for example:
```
curl "ENDPOINT_URL" --request POST \
   --header "TTL: 60" \
   --header "Content-Length: 0" \ 
   --header "Authorization: key=SERVER_KEY"
```

**Comment by jpmedley** [comment]
@nasearle I didn't notice it was Pete who assigned you until I reassigned to myself. Do you wa

**Comment by petele** [comment]
Please leave for Nick to fix

**Comment by nasearle** [comment]
Fixed in source doc. Will test again when we republish.

**Comment by petele** [comment]
republished
